subprojects/plugins/src/testFixtures/groovy/org/gradle/api/tasks/compile/AbstractCachedCompileIntegrationTest.groovy

```diff
@@ -59,6 +59,7 @@ abstract class AbstractCachedCompileIntegrationTest extends AbstractIntegrationS
         remoteProjectDir.deleteDir()
 
         when:
+        executer.requireOwnGradleUserHomeDir()
```

Comment:
By this I change the directory of the dependencies and test that the classpath is relocateable.